Add missing relation name validations for SWAP TABLE (#13381)
crate/crate: docs/appendices/release-notes/unreleased.rst

    );
    INSERT INTO t VALUES ('{"obj2": null, "target": "Sensor"}');
    SELECT * FROM t;

> - Fixed :ref:`swap table <alter_cluster_swap_table>` statements to throw an
>   exception when the table names provided contain ``.`` which is reserved as
>   the delimiter.

Reviewer: ```suggestion
- Fixed :ref:`swap table <alter_cluster_swap_table>` statements to throw an
  exception when the provided table names contain a ``.``.
```

What was the subsequent behavior? It would be easier for users to understand the implications if we describe what happened. 

"which is reserved as the delimiter." doesn't tell the user much. "The delimiter" isn't a well known thing in this context.

Author: Thank you, tried to be more descriptive.
